### PR TITLE
Feat: Add snackbar to show chapters reset was successful

### LINKED ISSUE #256: Add feedback when users reset completed chapters.
At present when users reset completed chapters, they can’t tell if the chapters have truly been reset since there is no feedback or  reaction for the action taken so they keep tapping and tapping till they go back to the 'Learn' tab and see that the chapters have actually been reset. We could add a flash message (pop-up) to let users know that their action has been reacted to. This will help improve user experience on the app.

**Screenshot:**WhatsApp Image 2023-03-06 at 10 32 03

### PR DESCRIPTION
**Pull request by Prakhar-Agarwal-byte**
Fix #256 When chapter reset button is clicked snackbar shows up to indicate success.

Screenshot_20230307_123811

### CODE REVIEW

File: app/src/main/java/com/goldenraven/padawanwallet/ui/settings/SettingsRootScreen.kt [hunk 1]
```diff
@@ -33,159 +37,167 @@ internal fun SettingsRootScreen(
...
+                    .fillMaxWidth()
+                    .padding(top = 8.dp, bottom = 40.dp, start = 40.dp, end = 40.dp),
+                color = Color(0xff8f8f8f),
+                thickness = 1.dp
             )
+
+            Button(
+                onClick = {
+                    viewModel.unsetAllCompleted()
+                    scope.launch { scaffoldState.snackbarHostState.showSnackbar("Chapters Reset Successful") }
```
Comment (thunderbiscuit) [inline app/src/main/java/com/goldenraven/padawanwallet/ui/settings/SettingsRootScreen.kt:186]: I tend to capitalize only the first letter in these small micro-copy items. Here that would be `Chapters reset successful`.

**Review by thunderbiscuit — CHANGES_REQUESTED** [review]
Awesome thanks for the PR. One small comment and this is ready to merge.

File: app/src/main/java/com/goldenraven/padawanwallet/ui/settings/SettingsRootScreen.kt (hunk 1 above)
Comment (Prakhar-Agarwal-byte) [reply app/src/main/java/com/goldenraven/padawanwallet/ui/settings/SettingsRootScreen.kt]: Updated it 👍

**Review by thunderbiscuit — APPROVED** [review]
ACK [c5be594](https://github.com/thunderbiscuit/padawan-wallet/pull/260/commits/c5be594a88e0713ff8e2cd543e34d893b70afddb).

**Pull request merged**